tap_postgres/db.py

```diff
@@ -49,6 +49,9 @@ def open_connection(conn_config, logical_replication=False):
         'connect_timeout': 30
     }
 
+    if conn_config.get('statement_timeout'):
```

Comment:
If we want to support `0`, for "unlimited", I think we need to explicitly check that this is not `None`, since `0` is falsey.